doc/api/async_hooks.md

```diff
@@ -228,7 +228,7 @@ closing it before the resource can be used. The following snippet demonstrates
 this.
 
 ```js
-require('net').createServer().listen(function() { this.close(); });
+require('net').createServer().listen(() => { this.close(); });
```

Comment:
This isn't correct because with this change `this` now refers to the parent scope (which is probably equal to `module.exports` in this case) instead of the server object.